[synthetics_global_variable] Add is_totp (#2832)
DataDog/terraform-provider-datadog: docs/resources/synthetics_global_variable.md

### Optional

- `description` (String) Description of the global variable.
- `is_totp` (Boolean) If set to true, the global variable is a TOTP variable. Defaults to `false`.
- `options` (Block List, Max: 1) Additional options for the variable, such as a MFA token. (see [below for nested schema](#nestedblock--options))
- `parse_test_id` (String) Id of the Synthetics test to use for a variable from test.
- `parse_test_options` (Block List, Max: 1) ID of the Synthetics test to use a source of the global variable value. (see [below for nested schema](#nestedblock--parse_test_options))
- `restricted_roles` (Set of String) A list of role identifiers to associate with the Synthetics global variable.

> - `secure` (Boolean) If set to true, the value of the global variable is hidden. Will be ignored if is_totp is true. Defaults to `false`.

Reviewer: ```suggestion
- `secure` (Boolean) If set to true, the value of the global variable is hidden. This setting is ignored if `is_totp` is set to `true`. Defaults to `false`.
```
